wp-config-pantheon.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+/**
+ * Set WP_ENVIRONMENT_TYPE according to the Pantheon Environment
+ */
+if (! defined('WP_ENVIRONMENT_TYPE')) {
```

Comment:
We should probably use getenv / putenv here.

If in the future we decided to move this logic to the prepend file, we could define WP_ENVIRONMENT in the prepend file, and that would effectively disable the code here, as intended. However, if we later decided to move this logic into environment variables, then the code here would take precedence over what we did in the platform.

While it's likely that the platform logic would be the same as what is here, we don't know if there might be some future need to do something different. So, it's a pretty minor point, but I think that putenv would be a little more flexible than define.

Reply:
Overriding an ENV variable is also an option for more bespoke workflows where re-setting a constant is not, so another reason for putenv 👍